driver/pelican/pelican.go

```diff
@@ -32,6 +32,7 @@ type Pelican struct {
 	password string
 	name     string
 	target   string
+	timezone string
```

Comment:
Let's make the type of this field `*time.Location` and use `time.LoadLocation` within the `DiscoverPelicans` logic. That way, we're not trying to parse the timezone descriptor every time we retrieve the thermostat's status.